extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -240,6 +240,11 @@ describe('BindExpression', () => {
       expect(evaluate('"abc".toUpperCase()')).to.equal('ABC');
     });
 
+    it('escaped quotes', () => {
```

Comment:
This also allows escaping of other chars.

Reply:
What else requires escaping?

Reply:
It'll be treated the same, but test to assert that would be good.